### DISCUSSION TITLE
How can I tell gatsbyImageData not to change aspect ratio?

### DISCUSSION THREAD

**Discussion by danielmoessner** [gatsbyjs/gatsby#30664] [Help]
I'm using the following options:
`gatsbyImageData(layout: FULL_WIDTH, placeholder: BLURRED)`

But it resizes my pictures to width of 1 and height of 0.75. How can I tell the API not to change my aspect ratio?

**Comment by rodneylab** [comment]
Hi Daniel,
The docs say aspect ratio should be maintained if you use FULL_WIDTH so this sounds weird.  Is the code open source? Let me know where it is so I can take a look if it is

**Reply by danielmoessner** [reply]
Yes it's a website project but I'll tell you how to get it run quickly:

git clone https://github.com/danielmoessner/recyclist.de.git
npm install
npm run dev
localhost:8000/___graphql

```
query Q {
  bikesYaml(slug: {eq: "benottomodelo500"}) {
    image4 {
      childImageSharp {
        gatsbyImageData(layout: FULL_WIDTH, placeholder: BLURRED)
        id
      }
    }
  }
}
```

Now you can take a look at the src and see that the image is not in its original aspect ratio.

**Reply by rodneylab** [reply]
Hi Daniel,
I see you have added
```
               transformOptions: { fit: CONTAIN }
```
to the queries in `/src/pages/fahrraeder.jsx`.  Has this fixed your issue?

**Reply by danielmoessner** [reply]
No it has not. I added this as a temporary fix, so that the whole image is shown.

**Reply by danielmoessner** [reply]
You can test with the example query I wrote above.

**Reply by danielmoessner** [reply]
I've created a repo with the problem reproduced:
https://github.com/danielmoessner/issue

I found out:
The metadata of the image is not correct. Can I tell gatsby-plugin-image not to use the metadata?

**Reply by lkshamus** [reply]
I have this same exact issue as you, @danielmoessner. Were you able to find a workaround?

**Reply by danielmoessner** [reply]
@lkshamus You can try imagemagick's auto-orient. But fixing this in gatsby-image should be really easy, as all the necessary information is in the metadata of the image. I also don't understand why the gatsby people don't accept it as an issue..

**Reply by danielmoessner** [reply]
I've described the problem in more detail here: https://github.com/gatsbyjs/gatsby/discussions/30703

I also pushed an example of the problem to netlify: https://gatsby-plugin-image-issue.netlify.app/
